app/code/Magento/GoogleAnalytics/Block/Ga.php

```diff
@@ -33,23 +34,31 @@ class Ga extends \Magento\Framework\View\Element\Template
      */
     private $cookieHelper;
 
+    /**
+     * @var Escaper
+     */
+    private $escaper;
+
     /**
      * @param \Magento\Framework\View\Element\Template\Context $context
      * @param \Magento\Sales\Model\ResourceModel\Order\CollectionFactory $salesOrderCollection
      * @param \Magento\GoogleAnalytics\Helper\Data $googleAnalyticsData
      * @param array $data
      * @param \Magento\Cookie\Helper\Cookie|null $cookieHelper
+     * @param Escaper $escaper
      */
     public function __construct(
         \Magento\Framework\View\Element\Template\Context $context,
         \Magento\Sales\Model\ResourceModel\Order\CollectionFactory $salesOrderCollection,
         \Magento\GoogleAnalytics\Helper\Data $googleAnalyticsData,
         array $data = [],
-        \Magento\Cookie\Helper\Cookie $cookieHelper = null
+        \Magento\Cookie\Helper\Cookie $cookieHelper = null,
+        Escaper $escaper
     ) {
         $this->_googleAnalyticsData = $googleAnalyticsData;
         $this->_salesOrderCollection = $salesOrderCollection;
         $this->cookieHelper = $cookieHelper ?: ObjectManager::getInstance()->get(\Magento\Cookie\Helper\Cookie::class);
+        $this->escaper = $escaper;
```

Comment:
So first thing of feedback. These new objects injected should be defaulting to null http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter